### DISCUSSION TITLE
LNBank wallet that is accessible from multiple users (or at least the server admin)?

### DISCUSSION THREAD

**Discussion by ishristov** [btcpayserver/btcpayserver#3660] [General]
From what I currently understand, when a user creates a LNBank wallet, that wallet is only accessible from that user. Even if another user has admin rights (or is the server admin), they cannot access or even see that LNBank wallet.

This can cause some problems if we have a server admin that helps (and hosts) multiple businesses to manager their stores.
The problem can arise if the server admin creates a new store and admin user for that store and instruct them how to setup their LNBank and configure the lightning payments to go to that LNBank wallet. If the server admin then goes to the LN config of that store, they cannot see the configured LNBank and if the server admin changes something, they cannot select and restore the LNbank config made by the store admin.

<img width="797" alt="Screenshot 2022-04-21 at 12 48 16" src="https://user-images.githubusercontent.com/1769047/164430235-392dc0db-ac73-4fc6-b7bd-aa58de639f11.png">

Would it make sense if the LNbank wallet can be configured and managed by multiple users or at least by the server admin as well? With the current setup, if the store admin users sets a LNBank wallet and for some reason the person loses access to that account (or maybe die?), the LNBank wallet will be lost because even the server admin cannot access that LNBank wallet.

**Comment by dennisreimann** [comment]
Yes, I recently got a similar request as well. Tracking that feature request over here: dennisreimann/btcpayserver#14
